### PR TITLE
fix #398 - Format address decoded from method arguments to appropriate prefix, format tip as balance

### LINKED ISSUE #398: Destination address is encoded with prefix 42 although I'm on Kusama
How to reproduce:

Send some funds from any address on Kusama CC2
- the `dest` address in the transaction details is always encoded with 42 prefix (starting with 5..) although it's on Kusama.

image

I suspect that it's not on Signer side, (couldn't find anything when trying to debug quickly). @yjkimjunior can you confirm? I'd send that to apps and the extension, I tested on both using Kusama encoding.

### PR DESCRIPTION
**Pull request by yjkimjunior**
Closes #398

IMG_0036

### CODE REVIEW

File: src/components/PayloadDetailsCard.js [hunk 1]
```diff
@@ -217,7 +233,9 @@ function ExtrinsicPart({ label, fallback, value }) {
             ? renderMethodDetails()
```
Comment (Tbaut) [inline src/components/PayloadDetailsCard.js]: I find this type of structure particularly hard to read, can we use the good old if/else?

Comment (yjkimjunior) [reply src/components/PayloadDetailsCard.js]: fair enough, i don't mind either way, ill chang eit

Comment (yjkimjunior) [reply src/components/PayloadDetailsCard.js]: sorry @Tbaut unfortunately I think I'll keep this as it is actually :/

**Comment by Tbaut** [comment]
Damn, I knew it. it's all right

**Pull request merged**